### ISSUE TITLE
How to add Spectral Normaliztion to CycleGAN ?

### ISSUE THREAD

**Issue by sappy5678** [junyanz/pytorch-CycleGAN-and-pix2pix#340]
When I try to add pytorch new feature, Spectral Normalization, to CycleGAN.
like this 

before
```python=
 sequence += [
    nn.Conv2d(ndf * nf_mult_prev, ndf * nf_mult,
        kernel_size=kw, stride=2, padding=padw, bias=use_bias),
    norm_layer(ndf * nf_mult),
    nn.LeakyReLU(0.2, True)
 ]
```

after
```
sequence += [
    spectral_norm(nn.Conv2d(ndf * nf_mult_prev, ndf * nf_mult,
        kernel_size=kw, stride=2, padding=padw, bias=use_bias)),
    nn.LeakyReLU(0.2, True)
]
```

but all of the image that CycleGAN generate are empty.
I don't know what's going wrong.
How can I solve that problem?
Thanks for your Help.

**Comment by SsnL** [comment]
Yep, LGTM too.
